package.json

```diff
@@ -31,22 +31,22 @@
   "dependencies": {
     "@babel/core": "^7.16.12",
     "@babel/eslint-parser": "^7.16.5",
-    "eslint-config-prettier": "^7.2.0",
-    "eslint-plugin-mocha": "^5.3.0",
-    "eslint-plugin-prettier": "^3.3.1",
-    "eslint-plugin-promise": "^5.1.0",
-    "eslint-plugin-rulesdir": "^0.1.0",
+    "eslint-config-prettier": "^8.5.0",
+    "eslint-plugin-mocha": "^10.1.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-promise": "^6.0.1",
+    "eslint-plugin-rulesdir": "^0.2.0",
     "eslint-plugin-sort-imports-requires": "^1.0.2",
     "eslint-plugin-sort-keys-fix": "^1.1.2",
     "eslint-plugin-sql-template": "^2.0.0"
   },
   "devDependencies": {
     "@uphold/github-changelog-generator": "^3.0.0",
-    "eslint": "~8.20.0",
-    "mocha": "^9.2.0",
+    "eslint": "~8.24.0",
+    "mocha": "^10.0.0",
     "pre-commit": "^1.2.2",
     "prettier": "^2.7.1",
-    "should": "^9.0.2"
+    "should": "^13.2.3"
   },
   "peerDependencies": {
     "eslint": "~8.20.0",
```

Comment:
```suggestion
    "eslint": "~8.24.0",
```

?

Reply:
That would be a braking change. Let’s do that on the next major